src/helper.cc

```diff
@@ -123,18 +123,14 @@ Helper::SessionBase::closeWritePipeSafely(const char * const id_name)
 }
 
 void
-Helper::SessionBase::dropQueued()
+Helper::SessionBase::dropQueued(Client &client)
 {
     while (!requests.empty()) {
         // XXX: re-schedule these on another helper?
```

Comment:
Supplying Client info to this method is inline with addressing that potential (and controversial) XXX. Most likely, this code will be eventually refactored to extract queued requests away from this dying session, but that refactoring is outside this PR scope, of course.